packages/plugin-ext/src/main/node/resolvers/local-file-plugin-deployer-resolver.ts

```diff
@@ -0,0 +1,34 @@
+/********************************************************************************
+ * Copyright (C) 2018 Red Hat, Inc. and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+
+import { PluginDeployerResolverContext } from '../../../common/plugin-protocol';
+import { injectable } from 'inversify';
+import * as path from 'path';
+import { LocalPluginDeployerResolver } from './local-plugin-deployer-resolver';
+
+@injectable()
+export class LocalFilePluginDeployerResolver extends LocalPluginDeployerResolver {
+    static LOCAL_FILE = 'local-file';
+
+    constructor() {
```

Comment:
`LocalFilePluginDeployerResolver` provides 0 arguments in the constructor while the base class `LocalPluginDeployerResolver` has 1 argument in its constructor.

I think this causes the problem in the CI.

Reply:
As mentioned, it is not related to CI, the check fails locally with the error: https://github.com/eclipse-theia/theia/pull/8745#pullrequestreview-542313320.
It can be reproduced by running:

```
npx run test @theia/example-browser
```

Reply:
Sorry folks, it seems I messed up something when rebasing this PR against the latest `master`. I actually already fixed these issues some time ago. I will take care and ping you again when I fixed this.

Reply:
@azatsarynnyy Thanks for your patience. CI is now green, and I also verified locally that Theia is starting properly.